packages/bsky/tests/feed-generation.test.ts

```diff
@@ -316,6 +328,17 @@ describe('feed generation', () => {
     })
   })
 
+  describe('getFeedSuggestions', () => {
+    it('returns list of suggested feed generators', async () => {
+      const resEven = await agent.api.app.bsky.feed.getFeedSuggestions(
+        {},
+        { headers: await network.serviceHeaders(sc.dids.bob) },
+      )
+      expect(forSnapshot(resEven.data)).toMatchSnapshot()
+      expect(resEven.data.feeds.map((fg) => fg.uri)).not.toContain(feedUriPrime) // taken-down
```

Comment:
lol hopefully we're not suggesting any taken down feeds 😅 

but probs good belt & suspenders